_maps/map_files/OmegaStation/OmegaStation.dmm

```diff
@@ -31739,273 +31173,1727 @@
+	master_tag = "incinerator_airlock_control";
+	pixel_y = 24
+	},
+/turf/open/floor/engine,
+/area/maintenance/disposal/incinerator)
+"rlq" = (
+/obj/machinery/door/firedoor,
+/obj/machinery/atmospherics/pipe/simple/supply/hidden,
+/obj/structure/cable/white{
+	icon_state = "1-2"
+	},
+/obj/effect/turf_decal/stripes/line{
+	dir = 2
+	},
+/obj/effect/turf_decal/stripes/line{
+	dir = 1
+	},
+/turf/open/floor/plasteel,
+/area/hallway/primary/aft)
+"rzn" = (
+/obj/machinery/status_display,
+/turf/closed/wall,
+/area/hallway/primary/starboard)
+"rzq" = (
+/obj/machinery/atmospherics/pipe/simple/scrubbers/visible{
+	dir = 6
+	},
+/turf/open/floor/plasteel,
+/area/engine/atmos)
+"rCw" = (
+/obj/machinery/door/window/brigdoor{
+	dir = 4;
+	name = "Creature Pen";
+	req_access_txt = "47"
+	},
+/obj/machinery/door/poddoor/preopen{
+	id = "xeno4";
+	name = "Creature Cell #4"
+	},
+/obj/structure/cable/white{
+	icon_state = "1-8"
+	},
+/obj/structure/cable/white{
+	icon_state = "2-8"
+	},
+/obj/effect/turf_decal/delivery,
+/turf/open/floor/plasteel/vault{
+	dir = 8
+	},
+/area/science/xenobiology)
+"rEx" = (
+/obj/machinery/atmospherics/pipe/simple/cyan/visible{
+	dir = 4
+	},
+/obj/structure/cable{
+	icon_state = "4-8"
+	},
+/obj/effect/turf_decal/stripes/line,
+/turf/open/floor/plasteel,
+/area/engine/atmos)
+"rVj" = (
+/obj/machinery/atmospherics/components/unary/vent_pump/siphon/on{
+	dir = 1;
+	frequency = 1441;
+	id_tag = "airmix_out";
+	name = "airmix vent"
+	},
+/turf/open/floor/engine/air,
+/area/engine/atmos)
+"rVs" = (
+/obj/machinery/atmospherics/components/unary/outlet_injector/on{
+	dir = 2;
+	frequency = 1441;
+	id = "mix_in";
+	name = "gas mix injector";
+	pixel_y = 1
+	},
+/turf/open/floor/engine/vacuum,
+/area/engine/atmos)
+"sak" = (
+/obj/effect/landmark/event_spawn,
+/turf/open/floor/plasteel/greenblue/side{
+	dir = 1
+	},
+/area/hydroponics)
+"sdL" = (
+/obj/effect/turf_decal/delivery,
+/obj/machinery/hydroponics/soil,
+/turf/open/floor/plasteel/vault/side,
+/area/maintenance/starboard/aft)
+"soC" = (
+/obj/machinery/atmospherics/pipe/simple/supply/hidden{
+	dir = 4
+	},
+/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden,
+/obj/structure/cable/white{
+	icon_state = "4-8"
+	},
+/turf/open/floor/plasteel/neutral/corner{
+	dir = 4
+	},
+/area/hallway/primary/starboard)
+"sqz" = (
+/turf/closed/wall,
+/area/hallway/primary/starboard)
+"srh" = (
+/obj/machinery/atmospherics/pipe/simple/green/visible{
+	dir = 4
+	},
+/obj/machinery/atmospherics/components/binary/pump{
+	name = "CO2 to Pure"
+	},
+/obj/effect/decal/cleanable/dirt,
+/turf/open/floor/plasteel/vault{
+	dir = 5
+	},
+/area/engine/atmos)
+"srR" = (
+/obj/machinery/atmospherics/components/unary/outlet_injector/on{
+	dir = 2;
+	frequency = 1441;
+	id = "n2o_in";
+	name = "n2o injector";
+	pixel_y = 1
+	},
+/turf/open/floor/engine/n2o,
+/area/engine/atmos)
+"sws" = (
+/obj/docking_port/stationary{
+	dheight = 1;
+	dir = 8;
+	dwidth = 12;
+	height = 17;
+	id = "syndicate_sw";
```

Comment:
uhhhhhhhhhhh

weren't these all supposed to be removed forever ago @ninjanomnom